server/services/transport/daily.mdx

```diff
@@ -916,6 +916,97 @@ See [update_subscriptions](https://reference-python.daily.co/api_reference.html#
 
 </ResponseField>
 
+### Permissions Management
```

Comment:
Ah cool! Had an issue already open for the work of documenting this: https://linear.app/dailyco/issue/CSDK-2684/canreceive-permission-pipecat-docs.

Reply:
Updated. Commit incoming.